rpc/src/cluster_tpu_info.rs

```diff
@@ -75,8 +75,8 @@ impl TpuInfo for ClusterTpuInfo {
     ) -> Vec<(&SocketAddr, Slot)> {
         let recorder = self.poh_recorder.read().unwrap();
         let leaders: Vec<_> = (0..max_count)
+            .rev()
             .filter_map(|future_slot| {
-                let future_slot = max_count.wrapping_sub(future_slot);
```

Comment:
`0..max_count` is expanded to [0, 1] usually because `max_count == 2` by default, and this reassigned `future_slot` ends up [2, 1].

This is different behavior from the counterpart called `get_leader_tpus`, which calls `.leader_and_slot_after_n_slots(0)`...

Reply:
Do you know why this line was added initially? in `fn get_leader_tpus` above we do not have safe maths, is it possible to replace it with a simple:
```
recorder.leader_and_slot_after_n_slots(i * NUM_CONSECUTIVE_LEADER_SLOTS)
```

Reply:
maybe, safe math is used because the new `get_leader_tpus_with_slots()` is added later. so, there's no particular reason.

that said, i'll leave these as-is to minimize diff.